galley/pkg/crd/validation/config.go

```diff
@@ -35,6 +35,8 @@ import (
 	"istio.io/istio/pkg/log"
 )
 
+var scope = log.RegisterScope("kube", "kube-specific debugging", 0)
```

Comment:
I think we should rename this to be "validation". Something different than the rest of the kube code. It is a different subsystem.